wonil/top_styles.css

```
@@ -0,0 +1,131 @@
+a{text-decoration: none; color: black;}
+.top_top{
+    display: grid;
+    grid-template-columns: 950px 200px;
+    grid-template-rows: 30px 60px;
+    gap: 10px;
+}
+.cut{color: gray; opacity: 0.2;}
+.bar_1{
+    grid-column: 2/3;
+    grid-row:1/2;
+    display: flex;
+    font-size: 12px;
+    float: right;
+    margin: 20px;
+}
+.register{color: rgb(95, 0, 128);}
+.cm_center_parent_ul{
+    list-style-type: none; 
+    padding: 0; 
+    margin: 0;
+}
+.cm_center_ul li a{
+    background-color: white; 
+    text-decoration: none; 
+    display: block;
+}
+.cm_center_ul{
+    list-style-type: none;
+    display: none; 
+    position: absolute;
+}
+.cm_center_parent_li:hover > .cm_center_ul{display: block;}
+.bar_2{
+    grid-column: 1/3;
+    grid-row:2/3;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    top: 100px; left: 100px;
+}
+.top_top_left{display: flex; gap: 20px; margin: 0 0 0 20px;}
+.kurly_link{display: flex; gap: 10px; align-items: center; font-size: 15px;}
+.market_kulry{color: rgb(95, 0, 128);}
+.beauty_kulry{opacity: 0.2;}
+.beauty_kulry:hover{opacity: 1.0;}
+.top_top_middle{
+    display: flex; align-items: center;
+    background-color: white;
+    border-style: solid;
+    border-width: 2px;
+    border-color: rgb(95, 0, 128);

```

```suggestion
    border: 2px solid rgb(95, 0, 128);
```
border은 3개의 값을 같이 사용해주는 게 일반적입니다.